models/lin_rate.h

```diff
@@ -0,0 +1,120 @@
+/*
+ *  lin_rate.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef LIN_RATE_H
+#define LIN_RATE_H
+
+// Includes from models:
+#include "rate_neuron_ipn.h"
+#include "rate_neuron_ipn_impl.h"
+#include "rate_neuron_opn.h"
+#include "rate_neuron_opn_impl.h"
+
+
+namespace nest
+{
+/* BeginDocumentation
+Name: lin_rate - Linear rate model
```

Comment:
was discussed in NEST developer meeting and decided to keep abbreviated model names as in other neuron models.